sequencer/src/bin/espresso-dev-node.rs

```diff
@@ -154,6 +159,10 @@ struct Args {
     )]
     max_block_size: u64,
 
+    /// The length of a hotshot epoch in hotshot blocks.
+    #[clap(long, env = "ESPRESSO_DEV_NODE_EPOCH_HEIGHT", default_value_t = 40)]
```

Comment:
40 as epoch height is okay for testing, but this needs to set when used by integrations team

Reply:
Do you expect it to break anything if we use 40?